doc/code/qml_io.rst

```diff
@@ -0,0 +1,15 @@
+qml.io
```

Comment:
Why this new file is needed?

Reply:
We made the decision to take the translation/interoperability functions and classes and put them inside a new folder `io`, so that's what that folder is -- it has `io.py` which contains all the translations for qiskit/qasm/etc... and also our `qualtran_io.py`.

Reply:
Will also need to update: 

https://github.com/PennyLaneAI/pennylane/blob/master/doc/index.rst

to get it to render in the API section.